scripts/update-spectrum-css.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { readFileSync, writeFileSync } from 'fs';
+import latestVersion from 'latest-version';
+import fg from 'fast-glob';
+
+async function update() {
+    let updated = false;
+    for (const packageJSONPath of await fg([
+        `./package.json`,
+        `./packages/**/package.json`,
+        `./projects/**/package.json`,
+        `./tools/**/package.json`,
+    ])) {
+        let shouldUpdate = false;
+        console.log(packageJSONPath);
+        const packageJSON = JSON.parse(readFileSync(packageJSONPath, 'utf-8'));
+        const { dependencies, devDependencies } = packageJSON;
+        for (const packageName in dependencies) {
+            if (packageName.startsWith('@spectrum-css')) {
+                const version = await latestVersion(packageName);
+                dependencies[packageName] = version;
+                shouldUpdate = true;
+            }
+        }
+        for (const packageName in devDependencies) {
+            if (packageName.startsWith('@spectrum-css')) {
+                const version = await latestVersion(packageName);
+                devDependencies[packageName] = version;
+                shouldUpdate = true;
+            }
+        }
+        if (shouldUpdate) {
+            packageJSON.dependencies = dependencies;
```

Comment:
Instead of manually writing the files here - would running `yarn upgrade` via an exec command be an option?

Reply:
No matter where I run `yarn upgrade` I can't get this to actually do what I want here. If this worked from the outside, we _might_ be able to get away without a script all together. Maybe it works in a newer version of `yarn`, but I'm not ready to dive into that rabbit hole at this moment. Are you able to use this across monorepo packages on your end?

Reply:
I have used it in the past yes but that's fair enough.  What issue are you seeing with it?  Does it not run at all or does it run in an unexpected way?